howso/hyperparameters.amlg

```diff
@@ -672,7 +672,6 @@
 						"auto_analyze_enabled" !autoAnalyzeEnabled
 						"analyze_threshold" !autoAnalyzeThreshold
 						"analyze_growth_factor" !autoAnalyzeGrowthFactorAmount
-						"auto_analyze_limit_size" !autoAnalyzeLimitSize
```

Comment:
I can't comment on the exact line, but should we change the `(if)` condition here to be `(if !autoAnalyzeEnabled ...`?

Reply:
functionally it's the same imo, but sure that's probably more readable